artcommon/artcommonlib/release_util.py

```diff
@@ -49,3 +51,26 @@ def isolate_el_version_in_release(release: str) -> Optional[int]:
     if el_suffix:
         return int(el_suffix[2:])
     return None
+
+
+def verify_advisory_status(advisory_info: Dict, editable: bool):
+    if editable:
+        if advisory_info["status"] in {"QE", "REL_PREP", "PUSH_READY", "IN_PUSH", "SHIPPED_LIVE"}:
+            raise VerificationError(f"Advisory {advisory_info['id']} should not be in {advisory_info['status']} state.")
+    else:
+        if advisory_info["status"] not in {"QE", "REL_PREP", "PUSH_READY", "IN_PUSH", "SHIPPED_LIVE"}:
+            raise VerificationError(f"Advisory {advisory_info['id']} should not be in {advisory_info['status']} state.")
+
+async def get_advisory_info(advisory: int, group, exectools, elliott_env_vars, working_dir) -> Dict:
+    cmd = [
```

Comment:
since this is artcommonlib let's not have a function here that wraps an elliott call
Let's call elliottlib.errata.get_raw_erratum and get whatever info we need